docs/_policies/workgroups.md

```diff
@@ -1,29 +1,22 @@
 # Workgroup Management
-## What is a Workgroup?
-
-The [Stanford Workgroup Manager](https://workgroup.stanford.edu/) is a web interface that gives Stanford community members a place to define a group of other community members based on their SUNet IDs for use in various applications. A workgroup is given a name that uniquely identifies it and succintly describes its purpose. Relevant to the Yens, the Workgroup Manager helps control access to individual [project space](/services/newProject.html) on ZFS (ex: `gsb-rc:[faculty-SUNet]-[projectname]`).
 
-## Adding/Removing Users to your Workgroup
-If you are an Administrator of a workgroup, you can use the Workgroup Manager interface to easily add users to and remove users from that workgroup. Adding users to a workgroup immediately grants them permissions to whatever resource that workgroup controls access to. Likewise, removing users revokes those permissions. You might want to do this if you no longer want someone to have access to your project space or as part of regular auditing of external collaborators you have sponsored access to the Yens for.
+## What is a Workgroup?
 
-Users can be added as Members and/or Administrators of a workgroup. While Members are simply granted permission to the resource controlled by the workgroup, Administrators have the power to add *other* users as Members/Administrators to the workgroup.
+The [Stanford Workgroup Manager](https://workgroup.stanford.edu/){:target="_blank"}, is a web interface that allows Stanford community members to define groups of other community members based on their SUNet IDs for use in various applications. Each workgroup is assigned a unique name that succintly describes its purpose. 
 
-!!! warning
-    Given that Administrators of a workgroup have the power to add other users, please be mindful of who you grant Administrator privileges to. It is *your responsibility* to audit both the Administrators and Members of your workgroup.
+Relevant to the Yens, the Workgroup Manager helps control access to individual [project space](/_policies/services){:target="_blank"} on ZFS (e.g., gsb-rc:[faculty-SUNet]-[projectname]). For example, if your project is stored at `/zfs/projects/faculty/jdoe-housing`, there will be a corresponding workgroup named `gsb-rc:jdoe-housing`.
```

Comment:
I think we can reword this to be more clear?

Workgroups on the Yens are used to manage system logins as well as access to [project space](/_policies/services){:target="_blank"}. Each project space has a corresponding workgroup, so members of the workgroup `gsb-rc:jdoe-housing` have read/write access to data stored at `/zfs/projects/faculty/jdoe-housing`. Members of these project workgroups will also have login access to the Yen servers more generally.